src/main/java/com/minecolonies/coremod/colony/IColony.java

```diff
@@ -73,4 +73,10 @@
      * @return an integer with a describing value.
      */
     int getLastContactInHours();
+
+    /**
+     * Get whether the colony can be auto deleted.
+     * @return true if so.
+     */
+    boolean getCanBeAutoDeleted();
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Rename this method to start with "is" or "has". [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS2047)